CHANGELOG.md

```diff
@@ -39,6 +39,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [David Airapetyan](https://github.com/davidair)
   [#4887](https://github.com/CocoaPods/CocoaPods/issues/4887)
 
+* Remove duplicate framework names  
```

Comment:
@abiligiri can this changelog entry be a bit more descriptive? Its just too generic, thankfully there is the link to the GH issue but still...